web/beacon-app/cypress/support/commands.ts

```diff
@@ -39,9 +39,17 @@
 
 import '@testing-library/cypress/add-commands';
 
-Cypress.Commands.add('loginWith', ({email, password}: any) =>
+Cypress.Commands.add('loginWith', ({email, password}) =>
   cy.visit('/')
   .get('input[name="email"]').type(email)
   .get('input[name="password"]').type(password)
-  .get('[data-testid="login-button"]').click();
+  .get('[data-testid="login-button"]').click()
 )
+
+declare global {
```

Comment:
This was added because I kept getting an error message. I used this link: https://docs.cypress.io/guides/tooling/typescript-support#Types-for-Custom-Commands